src/printer/utils.ts

```diff
@@ -110,6 +110,11 @@ export function isTextNodeEndingWithWhitespace(node: Node): node is TextNode {
 	return isTextNode(node) && endsWithWhitespace(getUnencodedText(node));
 }
 
+export function hasSetAttributes(node: TagLikeNode) {
```

Comment:
maybe this should be `hasSetDirectives` to be more clear? `hasSetNouns` reads to me as "this function is checking whether some Nouns are set in the variable."

Reply:
I agree!